content/articles/how-to-use-brotli-compression-method/index.md

```diff
@@ -0,0 +1,144 @@
+The browser sends an `Accept-Encoding` header with the algorithms it supports and its order of precedence; the server chooses one, uses it to compress the body of the `content-encoding` header to inform the browser of its choice. 
+- `Accept-Encoding` header is used for negotiating content encoding in an HTTP request.
+  example;
+  ```command
+  Accept-Encoding: gzip, deflate
+  ```
+- `content-encoding` response header lists any encodings that have been applied to the representation.
+  example;
+  ```command
+  Content-Encoding: gzip
+  ```
+Because content negotiation was used to select a representation based on its encoding, the server must include a vary header in the response that includes at least `Accept-encoding`; this allows caches to cache the various representations of the resource.
+
+
+### Application of Brotli
+There are two ways by which we can deliver Brotli compressed assets:
+1. Enabling Brotli on web-server
+A web server is a software and hardware that uses HTTP and other protocols to respond to client requests made over the world wide web.
+2. Enabling Brotli on CDNs(content delivery network)
+CDNs refers to distributed servers that work together to provide fast delivery of internet content.
+
+### Compression with Nginx statically
+Nginx is an open-source web server that can also work as a reverse proxy.
+Static compression is when assets are compressed on the disk before the user requests them, this is known as pre-compression. Compression is not perfomed when a user requests an asset. The asset is simply served from the disk after it has been compressed.
+Nginx has static compression capability for Brotli. Google has provided a [module](https://github.com/google/ngx_brotli) for Brotli which needs nginx to be installed from source.
+Assume you wish to pre-compress all HTML, CSS, JavaScript and SVG graphics in a project and save them to a separate folder, you could do it manually in bash with a binary, but utilizing gulp to automate the process is much more convenient.
+
+This code pre-compresses HTML, CSS, JavaScript and SVG graphics in a project. This is done via [module](https://github.com/google/ngx_brotli)
+```javaScript
+const brotlicompress = () => {
+    let src = "src/**/*.{html,js,css,svg}",
+        dest = "dist";
+
+    return gulp.src(src)
+        .pipe(brotli.compress({
+            extension: "br",
+            quality: 11
+        }))
+        .pipe(gulp.dest(dest));
+};
+
+exports.brotlicompress = brotlicompress;
+```
+After that, the brotliCompress task is called as follows:
+```bash
+gulp brotliCompress
+```
+All assets matched by the file glob(provided in the source variable) will be processed, and Brotli compresssed versions will be exported to the destination directory(specified in the dest variable). *Script.js* will become *scripts.js.br*, and *style.css* will become *style.css.br*
+
+This is how to specify Brotli-encoded assets to Apache web server;

```

Comment:
Why do you jump from nginx to apache? 

At least add proper heading.

The explanation for nginx was not enough